flink/flink-scala-parent/src/main/java/org/apache/zeppelin/flink/FlinkSqlInterrpeter.java

```diff
@@ -503,10 +503,16 @@ public void callSelect(String sql, InterpreterContext context) throws IOExceptio
   public abstract void callInnerSelect(String sql, InterpreterContext context) throws IOException;
 
   public void callSet(String key, String value, InterpreterContext context) throws IOException {
+    if (key.equals("execution.runtime-mode")) {
+      throw new IOException("execution.runtime-mode is not supported to set, " +
+              "you can use %flink.ssql & %flink.bsql to switch between streaming mode and batch mode");
+    }
```

Comment:
Only `IOException` is available here? `UnsupportedOperationException` seems be more suitable.

Reply:
That's right, thanks for the review, Fixed it